docs/user-guide/ingress.md

```diff
@@ -44,13 +44,7 @@ It can be configured to give services externally-reachable urls, load balance tr
 
 Before you start using the Ingress resource, there are a few things you should understand. The Ingress is a beta resource, not available in any Kubernetes release prior to 1.1. You need an Ingress controller to satisfy an Ingress, simply creating the resource will have no effect.
 
-On GCE/GKE there should be a [L7 cluster addon](https://github.com/kubernetes/contrib/blob/master/ingress/controllers/gce/README.md), deployed into the `kube-system` namespace:
-
-```shell
-$ kubectl get pods --namespace=kube-system -l k8s-app=glbc
-NAME                            READY     STATUS    RESTARTS   AGE
-l7-lb-controller-v0.6.0-chnan   2/2       Running   0          1d
-```
+GCE/GKE deploys a ingress controller on the master. You can deploy any number of custom Ingress controllers in pod(s) as long you annotate each ingress with the appropriate class, as indicated [here](https://github.com/kubernetes/contrib/tree/master/ingress/controllers/nginx#running-multiple-ingress-controllers) and [here](https://github.com/kubernetes/contrib/blob/master/ingress/controllers/gce/BETA_LIMITATIONS.md#disabling-glbc).
```

Comment:
But we aren't as strict as we should be, even about terms like replicas (which we use in a confusingly different context than the conventional meaning of data duplication)